builds/Resources/Include_RISCY_Config.mk

```diff
@@ -111,3 +122,26 @@ BSC_COMPILATION_FLAGS += \
 # +RTS -K1G -RTS " --bscflags=" -steps-max-intervals 200  -check-assert
 
 # ================================================================
+
+# ================================================================
+# Search path for bsc for .bsv files
+COREDIR ?= $(REPO)
+COREW_DIRS = $(COREDIR)/src_Core/Core:$(COREDIR)/src_Core/CPU:$(COREDIR)/src_Core/ISA:$(COREDIR)/src_Core/PLIC:$(COREDIR)/src_Core/Debug_Module:$(COREDIR)/src_Core/BSV_Additional_Libs:$(COREDIR)/src_Core/RISCY_OOO/procs/RV64G_OOO:$(COREDIR)/src_Core/RISCY_OOO/procs/lib:$(COREDIR)/src_Core/RISCY_OOO/coherence/src:$(COREDIR)/src_Core/RISCY_OOO/fpgautils/lib
```

Comment:
Can we unify whether we use `SOME_DIR` or `SOMEDIR`?

Reply:
I've gone with underscores.